spec/features/admin/admin_updates_user_info_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "rails_helper"
+
+feature 'admin updates a user', %{
+  As an admin
+  I want to be able to update user's information
+  so that I may better manage my application
+  } do
+    # Acceptance Criteria:
+    # [X] Updateable user information
+    # [X] notification when users information is updated
+    # [X] I must have admin to update users information
+
+  let(:admin) { FactoryGirl.create(:user, role: 'admin') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.